src/SessionManager.php

```diff
@@ -194,7 +194,7 @@ protected function initializeValidatorChain()
      */
     public function destroy(?array $options = null)
     {
-        if (headers_sent() || ! $this->sessionExists()) {
+        if (session_status() !== PHP_SESSION_ACTIVE) {
             return;
         }
```

Comment:
Let's add inline comment for the change since it is otherwise not enforced by test

```suggestion
        // session_destroy() requires active session while method $this->sessionExists() includes other conditions 
        if (session_status() !== PHP_SESSION_ACTIVE) {
```